bin/steal

```diff
@@ -1,179 +1,31 @@
-	.describe("bundle-steal", "Include steal.js in your main bundled JavaScript")
-	.describe("ignore", "For transform, a comma-separated list of modules to not include in the output")
-	.describe("o", "For transform, specify an output file")
-	.alias("o", "out")
-	.describe("source-maps", "Generate source maps")
-	.describe("source-maps-content", "Include the original source contents with the source maps")
-	.describe("minify", "Minify the output. Defaults to true except when used with --watch")
-	.describe("no-minify", "Do not minify the output")
-	.describe("w", "Watch for file changes and rebuild")
-	.alias("w", "watch")
-	.describe("live-reload-port", "Specify a port to use for the websocket server")
-	.default("live-reload-port", 8012)
-	.describe("verbose", "Verbose output")
-	.describe("quiet", "Quiet output")
-	.describe("cjs", "For export, it sets default +cjs output")
-	.describe("amd", "For export, it sets default +amd output")
-	.describe("global", "For export, it sets default +global-js and +global-css outputs")
-	.describe("all", "For export, it sets outputs to +cjs, +amd, +global-js, and +global-css")
-	.version(require("../package.json").version, "version")
+var path = require("path");
+var cmdBuild = require("../lib/cli/cmd_build");
+var cmdExport = require("../lib/cli/cmd_export");
+var cmdTransform = require("../lib/cli/cmd_transform");
+var cmdLiveReload = require("../lib/cli/cmd_live_reload");
+
+var argv = require("yargs")
+	.command(cmdBuild)
+	.command(cmdTransform)
+	.command(cmdLiveReload)
+	.command(cmdExport)
+	.version(function() {
+		return require("../package.json").version;
+	})
 	.help("h")
 	.alias("h", "help")
-	.check(function(argv){
-		var command = argv._[0];
-
-		if(!command) {
-			command = "build";
-		}
-
-		if(!_includes(commands, command)) {
-			throw "Unknown command: " + command;
-		}
-
-		if(command === "transform" && !argv.out) {
-			throw "When using transform, must specify an out file";
-		}
-
-	});
-
-// The command-line arguments
-var argv = yargs.argv;
-
-// Use package.json as the config by default.
-argv.config = argv.config;
-
-// Determine the location of the config file
-var config = argv.config[0] === "/" ?
-	argv.config : path.join(process.cwd(), argv.config);
-
-// Which command to run, either `build` or `transform`
-var command = argv._[0] || "build";
-
-// Load steal-tools
-var stealTools = require("../index");
-
-var system = { config: config };
-if(argv.main) {
-	system.main = argv.main;
-}
-
-var options = argv;
-
-switch (command) {
-	case "build":
-		return build();
-
-	case "transform":
-		return transform();
-
-	case "export":
-		return doExport();
-
-	case "live-reload":
-		doLiveReload(options);
-		break;
-}
+	.argv;
 
-function build() {
-	var promise = stealTools.build(system, options);
+// run `steal build` by default if no command was provided
+if (!argv._[0]) {
+	var spawn = require("child_process").spawn;
```

Comment:
I originally used cross-spawn-async here (and also replace the one in the test_cli file) but it brakes the build in 0.10.x because cross-spawn-async uses or depends on something that uses `Symbol`. Hopefully there is a way to workaround this limitation.